themes/default/languages/english/Validation.english.php

```diff
@@ -1,28 +1,28 @@
 <?php
 // Version: 1.1; Validation
 
-$txt['_validate_required'] = 'The %1$s field is required';
-$txt['_validate_valid_email'] = 'The %1$s field is required to be a valid email address';
-$txt['_validate_max_length'] = 'The %1$s field can hold a maximum of %2$s characters';
-$txt['_validate_min_length'] = 'The %1$s field needs to be a minimun of %2$s characters';
-$txt['_validate_length'] = 'The %1$s field needs to be exactly %2$s character in length';
-$txt['_validate_alpha'] = 'The %1$s field may only contain alpha characters (a-z)';
-$txt['_validate_alpha_numeric'] = 'The %1$s field may only contain alpha-numeric characters';
-$txt['_validate_alpha_dash'] = 'The %1$s field may only contain alpha characters &amp; dashes';
-$txt['_validate_numeric'] = 'The %1$s field may only contain numeric characters';
-$txt['_validate_integer'] = 'The %1$s field may only contain a numeric value';
-$txt['_validate_boolean'] = 'The %1$s field may only contain a true or false value';
-$txt['_validate_float'] = 'The %1$s field may only contain a float value';
-$txt['_validate_valid_url'] = 'The %1$s field is required to be a valid URL';
-$txt['_validate_url_exists'] = 'The URL %1$s does not exist';
-$txt['_validate_valid_ip'] = 'The %1$s field needs to contain a valid IP address';
-$txt['_validate_valid_ipv6'] = 'The %1$s field needs to contain a valid IPv6 address';
-$txt['_validate_contains'] = 'The %1$s field needs contain one of these values: %2$s';
-$txt['_validate_invalid_function'] = 'The specified validation function %1$s does not exist';
-$txt['_validate_without'] = 'The %1$s field may not contain the %2$s character';
-$txt['_validate_notequal'] = 'The %1$s field does not contain a valid value';
-$txt['_validate_isarray'] = 'The %1$s field does not contain a valid array';
-$txt['_validate_php_syntax'] = 'PHP syntax error: %2$s';
-$txt['_validate_limits'] = 'The %1$s field contains a value outside of allowable limits %2$s';
-$txt['_validate_generic'] = 'The %1$s field contains an invalid value';
-$txt['validation_failure'] = 'The form has the following error or errors that must be corrected before continuing:';
\ No newline at end of file
+$txt['_validate_required'] = 'The %1$s field is required.';
+$txt['_validate_valid_email'] = 'The %1$s field is required to be a valid email address.';
+$txt['_validate_max_length'] = 'The %1$s field can hold a maximum of %2$s characters.';
+$txt['_validate_min_length'] = 'The %1$s field needs to be a minimun of %2$s characters.';
+$txt['_validate_length'] = 'The %1$s field needs to be exactly %2$s character in length.';
+$txt['_validate_alpha'] = 'The %1$s field can only contain alpha characters (a-z).';
```

Comment:
If [a-z] is indeed all it takes it'd be better to say something like `lower case letters` or possibly `lower case letters without diacritics`.